console-timber-tree/src/main/java/com/jraska/console/timber/ConsoleTree.java

```diff
@@ -114,6 +122,11 @@ String getTag() {
     if (stackTrace.length <= CALL_STACK_INDEX) {
       return null;
     }
+    if (timeFormat != null) {
```

Comment:
This if should be part of `log` method. Timestamp is optional part of message, not a tag.